rollup.config.js

```diff
@@ -33,12 +33,15 @@ function getPlugins(standalone, transpile = true) {
 			)
 		}),
 		transpile && babel({
+			babelHelpers: 'runtime',
+			plugins: [
+				'@babel/plugin-transform-runtime'
```

Comment:
Already in preset

https://github.com/TrigenSoftware/babel-preset-trigen/blob/master/index.js#L98

Reply:
Ah, alrightee. Removed.

Reply:
Actually, I've added back now as it seems the build is erring otherwise. Though in my logging of babel-preset-trigen, I see it is getting the proper `true` value here, for some reason, it doesn't pick it up properly.

![image](https://user-images.githubusercontent.com/20234/81636429-6c129480-9446-11ea-9521-93bf76ed7136.png)

Reply:
And to be extra safe, it makes no difference whether there is a "json" extension or not.

Reply:
Let's add `skipPreflightCheck: true` as workaround to skip this error.

Reply:
Ok, I've added along with an explanatory comment linking to the rollup plugin issue you filed.